### PR TITLE
fix: correct file link

### PR DESCRIPTION
**Pull request by Netail**
Old URL resulting in 404

### CODE REVIEW

File: adding-sites.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ information about which sites Sherlock will query usernames for, and how that qu
 carried out.
 
 Here is an illustrative entry for one site in the 
-[data.json](https://github.com/sherlock-project/sherlock/blob/master/data.json) file.  Following this, is a
+[data.json](https://github.com/sherlock-project/sherlock/blob/master/sherlock/resources/data.json) file.  Following this, is a
```
Comment (ppfeister) [inline adding-sites.mdx:22]: Good catch.

That URL is soon to be invalid, though --- this one will remain active going forward

```suggestion
[data.json](https://github.com/sherlock-project/sherlock/blob/master/sherlock_project/resources/data.json) file.  Following this, is a
```

**Review by ppfeister — CHANGES_REQUESTED** [review]

File: adding-sites.mdx (hunk 1 above)
Comment (Netail) [reply adding-sites.mdx]: Ahh oke, basically copied the one above. I've updated both to the new one :)

**Review by ppfeister — APPROVED** [review]
LGTM

**Comment by Netail** [comment]
> LGTM

Awesome

**Pull request merged**